include/sys/ring_buffer.h

```diff
@@ -315,13 +315,14 @@ uint32_t ring_buf_put_claim(struct ring_buf *buf, uint8_t **data, uint32_t size)
  * Ring buffer instance should not mix byte access and item access
  * (calls prefixed with ring_buf_item_).
  *
- * @param  buf  Address of ring buffer.
- * @param  size Number of valid bytes in the allocated buffers.
+ * @param  buf     Address of ring buffer.
+ * @param  size    Number of valid bytes in the allocated buffers.
+ * @param  partial True if more data is expected to be put in the buffer.
```

Comment:
Would you mind elaborating here a bit more? Just to be clear that it does not allow us to put more data than allocated memory.